packages/lib/hooks/useMountedState.ts

```diff
@@ -9,13 +9,12 @@ import { useCallback, useEffect, useRef } from "react";
 export default function useMountedState(): () => boolean {
   const mountedRef = useRef<boolean>(false);
   const get = useCallback(() => mountedRef.current, []);
+  const [mount, setMount] = useState(false);
```

Comment:
Better name woud be mounted and setMounted

Reply:
makes sense, changes made